tests/fsharp/Compiler/Language/ByrefTests.fs

```diff
@@ -236,4 +236,30 @@ type MyClass() =
 
         CompilerAssert.Compile fsCmpl
         
-#endif
\ No newline at end of file
+#endif
```

Comment:
Not related to this change, but is the `#endif` here still relevant?

Reply:
Yes.